option optimize_for = LITE_RUNTIME;

message LimitedTimeBuyRequest
{
 required uint32 id = 1;
}
